gemonlinedb/xml/schema/VFAT3ChannelConfiguration.xsd

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:redefine schemaLocation="common.xsd">
+    <xs:simpleType name="ExtensionTableName">
+      <xs:restriction base="ExtensionTableName">
+        <xs:enumeration value="GEM_VFAT3_CHANNEL_CONF"/>
+      </xs:restriction>
+    </xs:simpleType>
+    <xs:simpleType name="KindOfPart">
+      <xs:restriction base="KindOfPart">
+        <xs:enumeration value="GEM VFAT3"/>
+      </xs:restriction>
+    </xs:simpleType>
+    <xs:complexType name="Data">
+      <xs:complexContent>
+        <xs:extension base="Data">
+          <xs:all>
+            <xs:element name="CHANNEL_REG_0" type="xs:unsignedShort"/>
```

Comment:
Can this be generalized?  This [xsd](https://github.com/cms-gem-daq-project/cmsgemos/blob/develop/gemhardware/xml/vfat/vfat_channel_register_types.xsd) shows a way to achieve this.

Reply:
OK... bummer

Reply:
Though, I suppose we could discuss how to leave the DB setup as is (`CHANNEL_REG_X`), but have all intermediate `XML`/objects stored/visualized as the individual bit fields, which would get validated against the `xsd`, and then the `createXMLForDBLoading` function runs the validation, and then dumps the `CHANNEL_REG_X` file.
Possibly not worth the effort, but let me know if you think it's worth pursuing

Reply:
I don't think having multiple XML formats is a good idea. On the C++ side, take a look at [`ChannelRegisterValue`](http://homepage.iihe.ac.be/~lmoureau/GEM/cmsgemos/doc/build/html/df/dd0/uniongem_1_1onlinedb_1_1_v_f_a_t3_channel_configuration_1_1_channel_register_value.html).